src/test/java/org/pesmypetcare/webservice/dao/communitymanager/ForumDaoTest.java

```diff
@@ -396,6 +404,45 @@ public void deleteMessage() throws DatabaseAccessException, DocumentException {
                     verify(batch).delete(same(documentReference));
                 }
 
+                @Test
+                public void reportMessage() throws DatabaseAccessException, DocumentException, InvalidOperationException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 122).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5633182)